### DISCUSSION TITLE
WINAPI Pragma

### DISCUSSION THREAD

**Discussion by nsgnkhibdk2cls0f** [modula3/cm3#1148] [General]
Example: https://github.com/modula3/cm3/blob/master/m3-ui/opengl/src/WIN32/GL.i3

Is it another way to state that the calling convention is STDCALL?

**Comment by 1997tn** [comment]
There are many other pragmas undocumented on pragmas.html, e.g: pragma <* PRAGMA UNALIGNED *> also used on WinAPI headers.

https://github.com/modula3/cm3/discussions/1136#discussioncomment-4690636
